src/app/server/Server.cpp

```diff
@@ -655,7 +655,11 @@ void Server::ResumeSubscriptions()
 Credentials::IgnoreCertificateValidityPeriodPolicy Server::sDefaultCertValidityPolicy;
 
 KvsPersistentStorageDelegate CommonCaseDeviceServerInitParams::sKvsPersistenStorageDelegate;
+#if CHIP_CRYPTO_PSA
+PSAOperationalKeystore CommonCaseDeviceServerInitParams::sPSAOperationalKeystore;
```

Comment:
Yes, that's right, but there are many `CommonCaseDeviceServerInitParams` initialized this way, and the `sPersistentStorageOperationalKeystore` has been used there already. I've just added another option to initialize `sPSAOperationalKeystore` instead of `sPersistentStorageOperationalKeystore` as `operationalKeystore` if the PSA backend is selected.

Reply:
Again, my question is: Are there many in-tree examples that use PSA here?

Reply:
Boris, I switched nRF Connect unit tests to PSA a few days ago. Also, OpenIOT runs tests with PSA. Apart from that, no in-tree examples use PSA as of now, but all nRF examples will switch to PSA within ~2 months.

Reply:
OK, so it's not like there are "many examples that would be hard to update to not use CommonCaseDeviceServerInitParams" right now?  And if nRF switches all examples, we should think about how to do that without proliferating the deprecated CommonCaseDeviceServerInitParams even more....

Reply:
It's not hard to not to update `CommonCaseDeviceServerInitParams` right now but I'm not sure in which direction we're planning to steer away from that before we switch nRF examples to PSA. If we remove `CommonCaseDeviceServerInitParams`, does it mean that every time we add a new thing to `ServerInitParams` we have to add the same boilerplate to each in-tree example? And have all device manufacturers read about interdependencies between the components, such as "when you use PSA crypto, you must not use an operational key store that writes keys to KVS" etc. The point of changing the default for PSA was that the existing default would not work at all with PSA due to a different key format.

Note that for things like crypto, most device manufacturers have no reason to provide their own crypto implementations. They just want to use whatever platform vendors like us recommend. I can imagine no better way of making everyone's life easier than to provide default implementations that were tested to work with each other. Everyone is free to tinker with the params once they learn Matter SDK and they need to.